index.html

```diff
@@ -0,0 +1,422 @@
+                      <li>My Page</li>
+                      <li>Logout</li>
+                </ul>
+            </nav>
+        </header>
+        
+        <header class="header-desk">
+            
+            <nav class="heads-all">
+                <ul>
+                    <li class="logo-img">
+                        <a href="./index.html">
+                            <img src="./img/CC_Global_Summit_2015 logo.png" alt="collision logo">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="./index.html"><h2>Home</h2></a>
+                    </li>
+                    <li>
+                        <a href="./about-me.html"><h2>About</h2></a>
+                    </li>
+                    <li>
+                        <a href="#"><h2>Program</h2></a>
+                    </li>
+                    <li>
+                        <a href="#"><h2>Join</h2></a>
+                    </li>
+                    <li>
+                        <a href="#"><h2>Sponsor</h2></a>
+                    </li>
+                    <li>
+                        <a href="#"><h2>News</h2></a>
+                    </li>
+                    <li class="campaign">
+                        <a href="#"><h2>CC Campaign</h2></a>
+                    </li>
+                </ul>
+            </nav>    
+        </header>   
+    <nav id=""    class="nav-bar">
+        <div class="hamburger">
+            <i class="fa fa-bars"></i>
+         </div>
+       <ul class="nav-menu">
+        <li class="close-btn"><i class="fa fa-times"></i></li>
+        <li class="nav-item">
+            <a href="/index.html" class="nav-link">Home</a>
+        </li>
+        <li class=" nav-item">
+            <a href="/about-me.html" class="nav-link">About</a>
+        </li>
+        
+       </ul>
+    </nav>
+</header>
+<section id="header" class="headline">
+    <div class="head-description">
+    <p id="first-p">"Face to Face with Gorilla's"</p>
+    <div class="main-heading">
+    <h1 class="first-h1">The global tours <br>
+        and travel agency <br> 
+         </h1>
+        </div>
+        <p class="description-text">We constantly tailor make and exceptional safari deals and exclusives 
+            for our clients and We’re proud to have a high number of loyal customers and 96% 
+            of our customers consider our service to be excellent.</p>
+
+    <h2 class="date-heading">2023.10.15(THU)~(16FRI)</h2>
+    <p id="year-p">@The Global tours and travel agency, Art Global Ag </p>
+</div>
+</section>
+<section id="main-program">
+    <div class="container">
+    <div class="programs">
+        <h2 class="main-pro">Main Programes</h2>
+        <hr class="line-h"/>
+    <div class="prog">
+        <div class="sub-cont">
+        <div class="imgcont">
+            <img src="./img/main-icon/Screenshot (207).png" alt="lecture">
+        </div>
+        <div class="text">
+            <span>lecture</span>
```

Comment:
- [ ] `required`
- [ ] Please make the border-boxes same size and and aligned like the origin template.
- [ ] please see screenshot below

`your template`
![image](https://user-images.githubusercontent.com/66230306/227655532-4f42df4a-2a6f-4f8e-963d-4bc4c7bc1865.png)

`orignal template`
![image](https://user-images.githubusercontent.com/66230306/227655849-58548f2f-c84e-4fe9-966d-3e75f3693e82.png)